packages/android-studio/android-studio.pacscript

```diff
@@ -2,33 +2,33 @@ name="android-studio"
 version="2021.1.1.20"
 url="https://dl.google.com/dl/android/studio/ide-zips/$version/android-studio-$version-linux.tar.gz"
 depends="libc6:i386 libncurses5:i386 libstdc++6:i386 lib32z1 libbz2-1.0:i386"
-description="Android Studio is the official integrated development environment (IDE) for Google's Android operating system, built on JetBrains' IntelliJ IDEA software and designed specifically for Android development."
+description="Android Studio is the official integrated development environment (IDE) for Google's Android operating system."
 hash="2f01b9203241169765b290e84cda5b8b5b6882b8b67ef10e1036e492262fac91"
 maintainer="Korkmatik <9c54efc1-143b-4f5f-9d2f-f6f189583f0f@aleeas.com>"
 prepare() {
     true
 }
 build() {
     #make needed directories
-    sudo mkdir -p "$STOWDIR/$name/opt/$name" "$STOWDIR/$name/usr/share/pixmaps" "$STOWDIR/$name/usr/share/applications" "$STOWDIR/$name/usr/bin"
+    sudo mkdir -p "$STOWDIR/$name/opt/$name" "$STOWDIR/$name/usr/share/pixmaps"
 }
 install() {
-    # Moving all android studio files to /opt
-    sudo cp -a bin lib jre plugins license LICENSE.txt build.txt product-info.json "$STOWDIR/$name/opt/$name"
+    # Moving android studio files to /opt
+    sudo cp -a bin lib jre plugins license LICENSE.txt build.txt product-info.json NOTICE.txt brokenPlugins.db icons.db classpath.txt "$STOWDIR/$name/opt/$name"
 
     # Creating a launcher in $PATH
-    sudo ln -s "$STOWDIR/$name/opt/$name/studio.sh" "$STOWDIR/$name/usr/bin/android-studio"
-
+    sudo ln -sf "$STOWDIR/$name/opt/$name/bin/studio.sh" "/usr/bin/android-studio"
```

Comment:
```suggestion
    sudo ln -sf "$STOWDIR/$name/opt/$name/bin/studio.sh" "$STOWDIR/$name/usr/bin/android-studio"
```

Reply:
I tried this but I get the error: source is an absolute symlink, and fails to install. No matter what I try I get this error except for a hard link, but that doesn't point to JDK or other android-studio files.

Reply:
How about moving `studio.sh` to `android-studio`?

Reply:
Same thing as hard-linking; can't find JDK or other android-studio files.